### PR TITLE
Added comments to GuildChannel.hx

### PR DESCRIPTION
**Pull request by Thresher25**
commented functions

### CODE REVIEW

File: src/com/raidandfade/haxicord/types/GuildChannel.hx [hunk 1]
```diff
@@ -30,26 +30,51 @@ class GuildChannel extends Channel {
...
+     */
     public function editChannel(cd,cb=null){
         client.endpoints.modifyChannel(id.id,cd,cb);
     }
-
+/**
+        Delete the given channel.
+        @param cb - Callback to send old channel to. Or null if result is not desired.
+     */
     public function deleteChannel(cb){
```
Comment (RaidAndFade) [inline src/com/raidandfade/haxicord/types/GuildChannel.hx]: Oops, can you change `cb` to `cb=null` here? Seems like I forgot 😢

**Comment by RaidAndFade** [comment]
Also it seems like you're still doing that thing where you're missing the first tab. Make sure you copy the whole comment from the beginning of the line, not from after the /**

File: src/com/raidandfade/haxicord/types/GuildChannel.hx [hunk 2]
```diff
@@ -38,30 +38,30 @@ class GuildChannel extends Channel {
...
         client.endpoints.modifyChannel(id.id,cd,cb);
     }
-/**
+    /**
         Delete the given channel.
         @param cb - Callback to send old channel to. Or null if result is not desired.
      */
     public function deleteChannel(cb){
-        client.endpoints.deleteChannel(id.id,cb);
+        cb=null;
```
Comment (RaidAndFade) [inline src/com/raidandfade/haxicord/types/GuildChannel.hx]: 🤦‍♀️

Comment (RaidAndFade) [reply src/com/raidandfade/haxicord/types/GuildChannel.hx]: the parameter sir, not the function itself.

File: src/com/raidandfade/haxicord/types/GuildChannel.hx [hunk 3]
```diff
@@ -58,8 +58,8 @@ class GuildChannel extends Channel {
         Delete the given channel.
         @param cb - Callback to send old channel to. Or null if result is not desired.
      */
-    public function deleteChannel(cb){
-        client.endpoints.deleteChannel(id.id,cb=null);
+    public function deleteChannel(cb=null){
+        client.endpoints.deleteChannel(id.id,cb);
```
Comment (RaidAndFade) [inline src/com/raidandfade/haxicord/types/GuildChannel.hx]: There ya go 😄

**Comment by Thresher25** [comment]
I think I got it, I hope

**Pull request merged**